smart-contracts/contracts/mixins/MixinGrantKeys.sol

```diff
@@ -35,6 +37,12 @@ contract MixinGrantKeys is
 
       _assignNewTokenId(toKey);
       _recordOwner(recipient, toKey.tokenId);
+
+      // Only assign the KeyManager for new or expired keys
+      if(toKey.expirationTimestamp <= block.timestamp) {
```

Comment:
Hmm are you sure?  It seems this puts the CC use case at risk.

Reply:
What's the risk you see here? 
What I want to avoid is this scenario:
- you have a valid key which you purchased, and for which you are the keyManager
- I (the keyGranter) come along and grant you a new key which extends your existing key, but I also set myself as the keyManager of your key, in effect stealing it from you.

Reply:
I buy with CC. I see a pending tx from Unlock protocol. I frontrun a crypto purchase. Then unlock buys and I cancel and refund, and chargeback - potentially doubling my investment

Reply:
No that's not exactly right.  I think we should just trust the keyGranter to do the correct thing here.  That's a trusted role - much more so than a keyManager of course.

e.g. if they have 1 day left on the key and they use Unlock CC to reup another month, we should have the ability to take the keyManager role to prevent chargeback issues.

Reply:
Since the keyGranter sets an expiration instead of adding time to it they already have permissions to revoke keys for all keyOwners without any refund. So this change does not seem to add more risk.

Reply:
hmm.  the keyGranter will be able to(as per our current permissions doc) expire any key, so I agree that this doesn't seem to pose any additional risk.